build.sbt

```diff
@@ -832,7 +832,7 @@ def commonScalacOptions(scalaVersion: String, isDotty: Boolean) =
                        "-language:existentials",
                        "-language:higherKinds",
                        "-language:implicitConversions",
-                       "-Ywarn-dead-code",
+                       //"-Ywarn-dead-code",
```

Comment:
Alas

Reply:
😬 yeah no idea how to work with `Nothing` without this

Reply:
Yeah I really spent a lot of time trying to get around it back when we first started this whole thing with the contravariant instances and just nothing doing.